### ISSUE TITLE
REQUEST: Use 2 wii remotes as ONE controller (like joycons)

### ISSUE THREAD

**Issue by leiburj** [ndeadly/MissionControl#123]
First of all, MC works perfectly in my setup, thankyou ndeadly for making this.

Onto my request: As the title says, would it be possible yo use two wii remotes without any extensions as a joint controller, the idea came to me while playing stern pinball arcade, mostly for the triggers, but it eould be useful for other games too. Thanks for everything.

**Comment by ndeadly** [comment]
This should be possible if the controllers advertise are able to themselves as left and right joycons. I am currently looking into being able to do just that. There are a few issues I need to solve first, but in theory once it's working you should be able to get the switch to merge them as one just by hitting L + R at the same time as you would on the real joycons. How they behave would be another question. Do you know if it's possible to remap the controls on 10.0.0 for joycons merged as a single controller?

**Comment by leiburj** [comment]
Yes, using MC im using a wiimote+nunchuck remapped, everything works, even the r and l analog to doad remapped joycons support this as well.

**Comment by leiburj** [comment]
Since wiimotes dont have l and r, something like a button combination (eg. Plus+1 on the wiimote to identify as joycon left and plus+2 to identify as right wgile syncing was my idea)

**Comment by ndeadly** [comment]
If that's the case then you should be able to map the wiimotes however you like once I have it working. Currently trying to figure out how the switch decides which orientation the joycon is being used in, as this causes issues for my mappings.

**Comment by leiburj** [comment]
There is an option on button remapping to config the joycons as horizontal or vertical here are some screenshots of the whole remapping process. This one is with the original joycons connected via BT but it works the same with all the controlles connected via MC.

![5A9858B0-3A91-4A3D-BCA4-A75ABF127AE5](https://user-images.githubusercontent.com/72759913/95779029-adca9400-0c9f-11eb-8cea-4abe0dbc82f1.jpeg)
![216A3588-9ED2-4E5E-818B-B3B40992F36F](https://user-images.githubusercontent.com/72759913/95779038-af945780-0c9f-11eb-9e89-d446f3a2f505.jpeg)
![8F2F6C42-C9D5-443C-9F34-AA2C7EA53463](https://user-images.githubusercontent.com/72759913/95779040-b02cee00-0c9f-11eb-9ab9-fea2163f0fdd.jpeg)
![2ECF2536-5DA7-4C9D-A4DE-64257DA683FB](https://user-images.githubusercontent.com/72759913/95779044-b0c58480-0c9f-11eb-95c1-be631c1a1d01.jpeg)
![91AEC736-9C2A-4AE6-9EF3-90E6E1FBFA19](https://user-images.githubusercontent.com/72759913/95779047-b1f6b180-0c9f-11eb-9a60-55aa369e1bbd.jpeg)
![61CA0914-8B9A-41FC-995E-127112BF68A6](https://user-images.githubusercontent.com/72759913/95779049-b28f4800-0c9f-11eb-89c2-864c61b37fd8.jpeg)
![2C060370-0CE2-4D4C-B60E-1693BDC6BD46](https://user-images.githubusercontent.com/72759913/95779051-b327de80-0c9f-11eb-9b97-365928aceeeb.jpeg)

**Comment by ndeadly** [comment]
So there's only one mapping to be applied regardless of whether the joycons are vertical, horizontal or joined?

**Comment by leiburj** [comment]
Yeah it works independently, the mapping is per controller. My guess is that the change/grip order screen is the one that sort of handles the configuration you use.

**Comment by leiburj** [comment]
As you see in the second to last screenshot, even if the joycon is horizontal you can configure the sl an sr buttons.

**Comment by leiburj** [comment]
Also you can save mappings, here is a screenshot of that. Also there is an option to change the stick orientation, il include that screenshot here too. I hope this helps you to save time by not having to code an these options that already work.
![35256DE5-362A-4771-88EA-4CCEC1CC6592](https://user-images.githubusercontent.com/72759913/95784021-677a3280-0ca9-11eb-9bc4-2a59f7668e28.jpeg)
![F3A3384B-3C59-4953-924E-6CD9D79A88B3](https://user-images.githubusercontent.com/72759913/95784024-6943f600-0ca9-11eb-9d15-74a502d03ba1.jpeg)
![4EBE1D72-947B-4AFB-86EF-494BE7937305](https://user-images.githubusercontent.com/72759913/95784025-69dc8c80-0ca9-11eb-9dd7-51171d4818ad.jpeg)
![E55624F9-700D-4FBC-890A-691B472DDC1F](https://user-images.githubusercontent.com/72759913/95784029-6c3ee680-0ca9-11eb-8c59-52e152077448.jpeg)
![A8EEB821-68D0-4156-92D2-CD3EF56F933A](https://user-images.githubusercontent.com/72759913/95784032-6d701380-0ca9-11eb-9bb1-9a733683ea1c.jpeg)
![D03F8126-699B-47AB-974C-2D44258EDF22](https://user-images.githubusercontent.com/72759913/95784035-6e08aa00-0ca9-11eb-94fa-87e176ab133f.jpeg)
![B35820DD-66D6-4AF6-9AE8-9FFCB625D656](https://user-images.githubusercontent.com/72759913/95784037-6ea14080-0ca9-11eb-8ca0-9d511c98eb9c.jpeg)
![6FC79CA5-6C3E-48D9-AC6D-AC44D2DF561E](https://user-images.githubusercontent.com/72759913/95784038-6f39d700-0ca9-11eb-8f39-38c19b9ed4fa.jpeg)

**Comment by leiburj** [comment]
As long as the controllers used by MC identify as OFFICIAL nintendo controllers, remapping is available, if they identify as a generic controller, remapping is NOT supported.
